dev_guide.md

```diff
@@ -2,19 +2,27 @@
 
 Dim Sum is a Spring Boot web app.
 
-* [Auto-configuration](https://docs.spring.io/spring-boot/docs/2.0.x/reference/html/using-boot-auto-configuration.html)
+- [Auto-configuration](https://docs.spring.io/spring-boot/docs/2.0.x/reference/html/using-boot-auto-configuration.html)
   automates a lot of application and library setup while allowing you to override any of its
   defaults
-* The [IoC container](https://docs.spring.io/spring-framework/docs/current/reference/html/core.html#beans)
+- The [IoC container](https://docs.spring.io/spring-framework/docs/current/reference/html/core.html#beans)
   simplifies the creation of application components via dependency injection
-* Controller classes respond to web requests
+- Controller classes respond to web requests
 
 ## Configuration
 
 App config is in `/resources/application.yml`. The properties in this file can be overridden
 using command-line parameters, system variables, or an external properties file. See the
 [README](/README.md) for more information on the external properties file.
 
+## Code Formatting
+
+Dim Sum has formatters configured for Java, HTML, and TypeScript in VSCode. For these to work, you
+must have the following extensions installed:
+
+- Language Support for Java(TM) by Red Hat
+- Prettier - Code Formatter
+
```

Comment:
Should `HTML Language Features` not be on here as well?

Reply:
I'm pretty sure that's built in and can't be installed/uninstalled?